src/mmw/js/src/core/opacityControl.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Consider adding a comment so that we remember that we modified this from the original.